apps/web/app/(app)/environments/[environmentId]/settings/(team)/billing/components/PricingTable.tsx

```diff
@@ -173,183 +120,105 @@ export const PricingTable = ({
-                </p>
-              </div>
+      <div className="mx-16 justify-between gap-4 rounded-lg capitalize">
+        <div className="flex w-full">
+          <h2 className="mr-2 inline-flex w-full text-2xl font-bold text-slate-700">
+            Current Plan: {team.billing.plan}
+            {cancellingOn && (
+              <Badge
+                className="mx-2"
+                text={`Cancelling: ${cancellingOn ? cancellingOn.toDateString() : ""}`}
+                size="normal"
+                type="warning"
+              />
+            )}
+          </h2>
+
+          {team.billing.stripeCustomerId && (
+            <div className="flex w-full justify-end">
+              <Button
+                size="sm"
+                variant="secondary"
+                className="justify-center py-2 shadow-sm"
+                loading={loadingCustomerPortal}
+                onClick={openCustomerPortal}>
+                {team.billing.plan !== "free" ? "Manage Subscription" : "Manage Card details"}
+              </Button>
             </div>
-          </>
-        )}
-
-        <PricingCard
-          title={"Formbricks Core & Surveys"}
-          subtitle={"Get 250 responses free every month"}
-          featureName={ProductFeatureKeys[ProductFeatureKeys.inAppSurvey]}
-          monthlyPrice={0}
-          actionText={"Starting at"}
-          team={team}
-          metric="responses"
-          sliderValue={responseCount}
-          sliderLimit={350}
-          freeTierLimit={appSurveyFreeResponses}
-          paidFeatures={coreAndWebAppSurveyFeatures.filter((feature) => {
-            if (team.billing.features.inAppSurvey.unlimited) {
-              return feature.unlimited !== false;
-            } else {
-              return feature.unlimited !== true;
-            }
-          })}
-          perMetricCharge={0.15}
-          loading={upgradingPlan}
-          onUpgrade={() => upgradePlan([StripePriceLookupKeys.inAppSurvey])}
-          onUnsubscribe={(e) => handleUnsubscribe(e, ProductFeatureKeys[ProductFeatureKeys.inAppSurvey])}
-        />
-
-        <PricingCard
-          title={"Link Survey Pro"}
-          subtitle={"Link Surveys include unlimited surveys and responses for free."}
-          featureName={ProductFeatureKeys[ProductFeatureKeys.linkSurvey]}
-          monthlyPrice={30}
-          actionText={""}
-          team={team}
-          paidFeatures={linkSurveysFeatures}
-          loading={upgradingPlan}
-          onUpgrade={() => upgradePlan([StripePriceLookupKeys.linkSurvey])}
-          onUnsubscribe={(e) => handleUnsubscribe(e, ProductFeatureKeys[ProductFeatureKeys.linkSurvey])}
-        />
+          )}
+        </div>
 
+        <div className="mt-2 rounded-lg border border-slate-300 bg-slate-100 py-4 capitalize shadow-sm dark:bg-slate-800">
+          <div className="mb-2 flex items-center gap-x-4"></div>
+          {
+            <div className="relative mx-8 mb-16 mt-4">
+              Responses
```

Comment:
Please wrap this text inside a html tag like the `p` tag and give it some color as well 🙏 pls do the same for all the text elements